scylladb/sphinx-scylladb-theme
# Set mascot opacity 1

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/238

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. The homepage mascot should not be transparent at all:

![image](https://user-images.githubusercontent.com/9107969/139036698-8bd8a1ab-8131-4233-8a6b-0b940f0bffb2.png)
